<ui version="4.0" >
 <class>ConnectionManager</class>
 <widget class="QDialog" name="ConnectionManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Connection Manager Window</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons.qrc" >:/icons/icon.svg</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="m_tabs" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_tabConnections" >
      <attribute name="title" >
       <string>Manage Connections</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Manage Connections</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QLabel" name="label_2" >
                <property name="text" >
                 <string>Connections:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListView" name="m_LstConnections" >
              <property name="selectionMode" >
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
              <property name="selectionBehavior" >
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="m_NewConnection" >
                <property name="text" >
                 <string>New</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m_DeleteConnection" >
                <property name="text" >
                 <string>Delete</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" >
              <item row="1" column="0" >
               <widget class="QLabel" name="label_7" >
                <property name="text" >
                 <string>Host Address:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" >
               <widget class="QLineEdit" name="m_HostAddress" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="1" column="2" >
               <widget class="QLabel" name="label_8" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Port:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3" >
               <widget class="QLineEdit" name="m_HostPort" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item row="2" column="0" >
               <widget class="QLabel" name="label_9" >
                <property name="text" >
                 <string>User Name:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="3" >
               <widget class="QLineEdit" name="m_UserName" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0" >
               <widget class="QLabel" name="label_10" >
                <property name="text" >
                 <string>Password:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1" colspan="3" >
               <widget class="QLineEdit" name="m_Password" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
                <property name="echoMode" >
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item row="4" column="0" >
               <widget class="QLabel" name="label_11" >
                <property name="text" >
                 <string>Private Key:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1" colspan="3" >
               <widget class="QLineEdit" name="m_PrivateKey" >
                <property name="enabled" >
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>171</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QVBoxLayout" >
                <item>
                 <widget class="QPushButton" name="m_Advanced" >
                  <property name="text" >
                   <string>Advanced</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer>
                  <property name="orientation" >
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" >
                   <size>
                    <width>102</width>
                    <height>31</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Manage Groups</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Manage Groups</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QComboBox" name="m_GroupName" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="editable" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m_NewGroup" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>New</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m_DeleteGroup" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Delete</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <widget class="QLabel" name="label_3" >
                <property name="text" >
                 <string>Connection in group:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListView" name="m_LstConsInGroup" />
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="m_AddToGroup" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../icons.qrc" >:/icons/arrow_left.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_RemoveFromGroup" >
              <property name="text" >
               <string/>
              </property>
              <property name="icon" >
               <iconset resource="../icons.qrc" >:/icons/arrow_right.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>20</width>
                <height>31</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <widget class="QLabel" name="label_4" >
                <property name="text" >
                 <string>Available Connections:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListView" name="m_LstAvailableConnections" />
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_OK" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc" />
 </resources>
 <connections/>
</ui>
